Assets/Scripts/LoudObjects/LoudObject.cs

```diff
@@ -0,0 +1,90 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class LoudObject : MonoBehaviour
+{
+    public float thrust;
+    public Vector3 direction;
+    public float dropRadius;
+    public float innerDropRadius;
+    public float loudObjectRadius;
+    private float distance;
+    private Rigidbody rb;
+
+    public float minShake;
+    public float maxShake;
+    Renderer[] rends;
+    
+
+    void Start()
+    {
+        rb = GetComponent<Rigidbody>();
+        rends = GetComponents<Renderer> ();
+    }
+
+    void Update()
+    {
+        distance = Vector3.Distance(transform.position, GameManager.Instance.GetPlayerTransform().position);
+        if (distance <= innerDropRadius)
+        {
+            rb.AddForce(direction * thrust);
+            rb.useGravity = true;
+            NotifyChasingNurse();
+        } 
+        else if (distance <= (innerDropRadius + 3))

```

Comment:
Probably don't need the +3 here? We can just set the innerDropRadius accordingly.

Reply:
So I have three radius zones, outer zone starts the vibration using LERP and inner zone vibrates at a constant max vibration and the final innerDropRadius is when the plates actually move. I can switch it also to the other one, but the max vibration is hard to see because the plates dropping is pretty instantaneous! What do you think?

Reply:
Mmm it might be better and more clear if we did something like (this is untested so it may need to be changed):
```
float clampedDistance = Mathf.Clamp(distance, innerDropRadius, dropRadius);
float lerpedDistance = Mathf.Lerp(0f, 1f, (clampedDistance - innerDropRadius) / (dropRadius - innerDropRadius));
float dynamicShake = Mathf.Lerp(maxShake, minShake, lerpedDistance);
```

This should cover everything except for the first if (`if (distance <= innerDropRadius)`) I believe? Since it covers when outside the largest radius (because it'll just be `minShake`, which can be 0f if we want), and when inside the innerDropRadius because it'll go to `maxShake`?

Reply:
I just updated with that!

Reply:
It worrks! it definitely gets way closer to max shake than mine did :) thanks!